tests/samples/songkick/elysianfields.json

```diff
@@ -230,16 +230,19 @@
                 }
             ],
             "http://ogp.me/ns#description": [
+                {
+                    "@value": ""
+                },
                 {
                     "@value": "Buy tickets for an upcoming Elysian Fields concert near you. List of all Elysian Fields tickets and tour dates for 2017."
                 }
             ],
             "http://ogp.me/ns#image": [
                 {
-                    "@value": "http://images.sk-static.com/images/media/img/col4/20100330-103600-169450.jpg"
+                    "@value": "http://images.sk-static.com/SECONDARY_IMAGE.jpg"
```

Comment:
Changing this order will create a bug in function `test_rdfa_not_preserving_order`. Why do you have to change the order here?

Reply:
I guess the reason is that it is because you are comparing with the sorting values... could you sort the values in the test function instead? That way `test_rdfa_not_preserving_order` would keep working as xfail case.

Reply:
I'm trying to sort the list in this failing function to fix it. I'm almost there.

Reply:
@croqaz the function `test_rdfa_not_preserving_order` should be preserved as it is. It is an `xfail` function, which means it is a test that should be passing, but it is not. So don't sort on this function, sort it in `test_all`. 

Probably you understood well, but just in case :-)

Reply:
I see. Well, to fix the test_rdfa_not_preserving_order() all we need to do is:

```py
for rdf in data['rdfa']:
            for key, pairs in rdf.items():
                if ':' in key and isinstance(pairs, list):
                    rdf[key] = sorted(pairs, key=lambda e: e["@value"])
        self.assertEqual(jsonize_dict(data)['rdfa'], expected['rdfa'])
```

Reply:
`test_rdfa_not_preserving_order()` fails randomly as it is exactly now, with my changes or not, because the order of the duplicate values is random.
So it doesn't matter if the duplicate values are sorted correctly or not in the JSON file.
The function is still a good way to replicate the bug.

Reply:
I see. Anyway, still, the order of the values for this case in `elysianfields.json` should be reversed. The first image should be `http://images.sk-static.com/images/media/img/col4/20100330-103600-169450.jpg` and the second image should be `http://images.sk-static.com/SECONDARY_IMAGE.jpg` as it was before, because this is the appearance order in the page.

This change will require some more code in the function `test_all` so that test passes, but this is the correct way. Otherwise `test_rdfa_not_preserving_order` would be incorrect, so wouldn't represent properly a case that should pass but is failing. (with current PR code `test_rdfa_not_preserving_order` represents a case that is failing and would also fail even in case #116 is fixed)

Reply:
I see your point. I'm working on it.

Reply:
Ok, this is done, I forgot to mention.
All the meta tags are converted to JSON objects and the sorting of JSON in the tests matches the actual HTML.